oam/install_scripts/columnstore-post-install.in

```diff
@@ -261,16 +261,13 @@ if [ ! -z "$MCS_USE_S3_STORAGE" ] && [ $MCS_USE_S3_STORAGE -eq 1 ]; then
   if [ -z "$MCS_S3_BUCKET" ]; then
     echo "Environmental variable \$MCS_USE_S3_STORAGE is set but there is no \$MCS_S3_BUCKET."
   fi  
-  if [ -z "$MCS_S3_ENDPOINT" ]; then
-    echo "Environmental variable \$MCS_USE_S3_STORAGE is set but there is no \$MCS_S3_ENDPOINT."
-  fi  
   if [ -z "$MCS_S3_ACCESS_KEY_ID" ]; then
     echo "Environmental variable \$MCS_USE_S3_STORAGE is set but there is no \$MCS_S3_ACCESS_KEY_ID."
   fi  
   if [ -z "$MCS_S3_SECRET_ACCESS_KEY" ]; then
     echo "Environmental variable \$MCS_USE_S3_STORAGE is set but there is no \$MCS_S3_SECRET_ACCESS_KEY."
   fi  
-  if [ -z "$MCS_S3_BUCKET" ] || [ -z "$MCS_S3_ENDPOINT" ] || [ -z "$MCS_S3_ACCESS_KEY_ID" ] || [ -z "$MCS_S3_SECRET_ACCESS_KEY" ]; then 
+  if [ -z "$MCS_S3_BUCKET" ] || [ -z "$MCS_S3_ACCESS_KEY_ID" ] || [ -z "$MCS_S3_SECRET_ACCESS_KEY" ]; then 
```

Comment:
Does SM use a default Endpoint when omitted from storagemanager.cnf?

Reply:
Empty endpoint works for AWS however I couldn't set it empty so that was the main reason of this change.